src/index.js

```diff
@@ -134,24 +141,34 @@ export default function(options = {}) {
 
               if (getExport) {
                 codeExportDefault = getExport(result.opts.from)
-                Object.keys(codeExportDefault).forEach(k => {
-                  const camelCasedKey = dashesCamelCase(k)
-                  if (reserved.check(camelCasedKey)) {
-                    console.warn(
-                      chalk.yellow('You are using a reserved keyword'),
-                      chalk.cyan(camelCasedKey),
-                      chalk.yellow(
-                        "as className so it's not available in named exports"
+
+                if (escapeDashes) {
+                  const escapedExportDefault = {}
+                  Object.keys(codeExportDefault).forEach(k => {
+                    const camelCasedKey = dashesCamelCase(k)
+                    escapedExportDefault[camelCasedKey] = codeExportDefault[k]
+                  })
+                  codeExportDefault = escapedExportDefault
+                }
+                if (exportNamed) {
```

Comment:
After some reflection I don't see the true value of having `escapeDashes`as an optional parameter here.

- when we have named exports, the escaping is mandatory
- when we don't use the named exports, then everything is exported inside of an object which can be later accessed in js by calling `style.className` or `style['class-Name']`, so we don't need any escaping here, thus keeping the css' structure/naming intact as it should be

So this pr actually it does the same thing as when providing/not providing the `getExport` function in the params of the plugin.